### PR TITLE
Use LogOutput to replace otErr: Wallet.cpp

### PR DESCRIPTION
**Pull request by Wallydog66**
Passes all unit tests.

### CODE REVIEW

File: src/api/Wallet.cpp [hunk 1]
```diff
@@ -1968,8 +1972,8 @@ ConstServerContract Wallet::Server(const proto::ServerContract& contract) const
         if (candidate) {
             if (candidate->Validate()) {
                 if (serverID.get() != candidate->ID()) {
-                    otErr << OT_METHOD << __FUNCTION__ << ": Wrong contract id"
-                          << std::endl;
+                    LogOutput(OT_METHOD)(__FUNCTION__)(": Wrong contract id.")
```
Comment (tommyodom) [inline src/api/Wallet.cpp]: Capitalize "id"

File: src/api/Wallet.cpp [hunk 2]
```diff
@@ -1941,16 +1945,16 @@ ConstServerContract Wallet::Server(const proto::ServerContract& contract) const
...
+            .Flush();
 
         return {};
     }
 
     const auto nymID = Identifier::Factory(contract.nymid());
 
     if (nymID->empty()) {
-        otErr << OT_METHOD << __FUNCTION__ << ": Invalid nym id" << std::endl;
+        LogOutput(OT_METHOD)(__FUNCTION__)(": Invalid nym id.").Flush();
```
Comment (tommyodom) [inline src/api/Wallet.cpp]: Capitalize "id"

File: src/api/Wallet.cpp [hunk 3]
```diff
@@ -981,7 +978,7 @@ ConstNym Wallet::Nym(const proto::CredentialIndex& serialized) const
     const auto nymID = Identifier::Factory(id);
 
     if (nymID->empty()) {
-        otErr << OT_METHOD << __FUNCTION__ << ": Invalid nym id" << std::endl;
+        LogOutput(OT_METHOD)(__FUNCTION__)(": Invalid nym id.").Flush();
```
Comment (tommyodom) [inline src/api/Wallet.cpp]: Capitalize "id"

**Comment by Wallydog66** [comment]
All recommended changes have been made

**Comment by FellowTraveler** [comment]
This one looks good to me, you can squash it with all the others.

**Pull request closed without merging**